styles/styles.css

```diff
@@ -0,0 +1,7 @@
+.text-yellow {
```

Comment:
Tried using tailwind classes for color and margin, but they dont seem to work. 

They do work for font and text size however 🤷‍♂️